lib/api/src/tests/stories.test.js

```diff
@@ -531,49 +532,31 @@ describe('stories API', () => {
       expect(initialStoriesHash['a--1'].args).toEqual({ a: 'b' });
       expect(initialStoriesHash['b--1'].args).toEqual({ x: 'y' });
 
-      const api = new EventEmitter();
-      init({ api });
-
+      init();
       api.emit(STORY_ARGS_UPDATED, 'a--1', { foo: 'bar' });
+
       const { storiesHash: changedStoriesHash } = store.getState();
-      expect(changedStoriesHash['a--1']).toEqual({
-        isLeaf: true,
-        parent: 'a',
-        kind: 'a',
-        name: '1',
-        parameters: {},
-        path: 'a--1',
-        id: 'a--1',
-        args: { foo: 'bar' },
-      });
-      expect(changedStoriesHash['b--1']).toEqual({
-        isLeaf: true,
-        parent: 'b',
-        kind: 'b',
-        name: '1',
-        parameters: {},
-        path: 'b--1',
-        id: 'b--1',
-        args: { x: 'y' },
-      });
+      expect(changedStoriesHash['a--1'].args).toEqual({ foo: 'bar' });
```

Comment:
REMOVED some data is the equals as the only thing we care about in THIS test is the args property